src/components/data_comps/datm/cime_config/namelist_definition_datm.xml

```diff
@@ -285,6 +290,7 @@
       <value stream="CLMCRUNCEP">domain.lnd.360x720.130305.nc</value>
       <value stream="CLMGSWP3v1">domain.lnd.360x720_gswp3.0v1.c170606.nc</value>
       <value stream="CLMGSWP3">domain.lnd.360x720_gswp3.0v1.c170606.nc</value>
+      <value stream="CLMNLDAS">domain.lnd.nldas2_0224x0464_c110415.nc</value>
```

Comment:
IS the NLDAS domain really different than the NLDAS2 domain? It is the same size. Also we decided that 0.125nldas2_0.125nldas2 was a more descriptive name rather than the grid size as it gives the grid cell resolution. You can query the file easy enough to get the number of grid cells.

Reply:
@jonbob ^

Reply:
Again, our PR was just to get a lost capability back into the system. I don't know much about the specifics....